configdb/hardware/tests.py

```diff
@@ -61,13 +62,23 @@ def test_write_instrument(self):
         self.assertEqual(saved_instrument.telescope.code, self.telescope.code)
 
     def test_patch_instrument(self):
-        self.assertEqual(self.instrument.state, Instrument.DISABLED)
+        self.assertEqual(self.instrument.state, 'DISABLED')
 
         self.client.login(username='tst_user', password='tst_pass')
         self.client.patch('/instruments/{}/'.format(self.instrument.pk), json.dumps({'state': 'MANUAL'}),
                           content_type='application/json')
         self.instrument.refresh_from_db()
-        self.assertEqual(self.instrument.state, Instrument.MANUAL)
```

Comment:
I kind of like having the `Instrument.MANUAL` enum-like states defined in the Instrument model. As I have it now, they're just gonna be strings but I am happy to make it work like an enum too. Opinions?

Reply:
Either way is fine. I agree enums look a little nicer and prevent you from using magic strings in tests and stuff.